core/wiki/macros/copy-to-clipboard.tid

```diff
@@ -1,6 +1,8 @@
 title: $:/core/macros/copy-to-clipboard
 tags: $:/tags/Macro
 
+<!-- These are possibly now deprecated in prefrence to the action widget -->
```

Comment:
We need to decide if they are deprecated. If we decide they are we need to add some info to the docs and provide new examples with the new "best practice"